elasticsearch.yml

```diff
@@ -1,5 +1,6 @@
 production: &default
   base_uri: <%= ENV["ELASTICSEARCH_URI"] || 'http://localhost:9200' %>
+  base_uri_search_b: <%= ENV["ELASTICSEARCH_B_URI"] %>
```

Comment:
Could these two variables have more similar names?  Why not just `base_uri_b`?

Reply:
Yeah good point. Changed.